### PR TITLE
API для сущностей

### PR DESCRIPTION
**Pull request by marodru**
Большой пласт работы для того чтобы сделать yeti платформой с полнофункциональным API.
Там попал некоторый мусор про CI и bintray правда :(

### CODE REVIEW

**Comment by dmitry-sinina** [comment]
Seems Audit log broken - after data modification whole objetc AdminUser serialized to database.

```
yeti=# SELECT whodunnit from gui.versions order by id desc limit 3;
           whodunnit           
-------------------------------
 #<AdminUser:0x00562a0fbb7ee0>
 #<AdminUser:0x00562a0b6830e0>
 3
(3 rows)
```

audit_log

**Comment by dmitry-sinina** [comment]
@marodru Seems cdr/49_50.sql migration is useless - is it modificating old stored procedure. In 48_49.sql new one was implemented(i_versions argument was added).

**Comment by dmitry-sinina** [comment]
Another problem - system api authorization(I mean /api/rest/system). Seems authorization was enabled globally. System api used for internal purposes and should work without authorization - for example we use curl to run some periodical tasks:
```
MAILTO=""
10 * * * *   yeti-web curl -X PUT -H 'Content-Length: 0' http://127.0.0.1:6666/api/rest/system/jobs/CdrPartitioning/run
*/30 * * * * yeti-web curl -X PUT -H 'Content-Length: 0' http://127.0.0.1:6666/api/rest/system/jobs/CdrBatchCleaner/run
20 * * * *   yeti-web curl -X PUT -H 'Content-Length: 0' http://127.0.0.1:6666/api/rest/system/jobs/CdrArchiving/run
* * * * *    yeti-web curl -X PUT -H 'Content-Length: 0' http://127.0.0.1:6666/api/rest/system/jobs/EventProcessor/run
* * * * *    yeti-web curl -X PUT -H 'Content-Length: 0' http://127.0.0.1:6666/api/rest/system/jobs/CallsMonitoring/run
35 * * * *   yeti-web curl -X PUT -H 'Content-Length: 0' http://127.0.0.1:6666/api/rest/system/jobs/StatsClean/run
25 * * * *   yeti-web curl -X PUT -H 'Content-Length: 0' http://127.0.0.1:6666/api/rest/system/jobs/StatsAggregation/run
30 5 * * *   yeti-web curl -X PUT -H 'Content-Length: 0' http://127.0.0.1:6666/api/rest/system/jobs/Invoice/run
*/15 * * * * yeti-web curl -X PUT -H 'Content-Length: 0' http://127.0.0.1:6666/api/rest/system/jobs/ReportScheduler/run
*/16 * * * * yeti-web curl -X PUT -H 'Content-Length: 0' Applitcation responds 401 on requests to system API.
http://127.0.0.1:6666/api/rest/system/jobs/TerminationQualityCheck/run
* * * * * yeti-web curl -X PUT -H 'Content-Length: 0' http://127.0.0.1:6666/api/rest/system/jobs/DialpeerRatesApply/run
```

File: debian/yeti-web.yeti-cdr-billing@.service [hunk 1]
```diff
@@ -26,9 +26,11 @@ Environment=GEM_HOME=/home/yeti-web/.gem
 Environment=GEM_PATH=/home/yeti-web/.gem:/home/yeti-web/.bundle_gem
 
 WorkingDirectory=/home/yeti-web/pgq-processors
-RuntimeDirectory=yeti-cdr-billing
-ExecStart=/usr/bin/ruby /home/yeti-web/.gem/bin/bundle exec rake pgq:worker processor=%i pid_file=/run/yeti-cdr-billing/%i.pid config_file=/home/yeti-web/config/cdr_billing.yml
-PIDFile=/run/yeti-cdr-billing/%i.pid
+
+RuntimeDirectory=yeti-cdr-billing-%i
+ExecStart=/usr/bin/ruby /home/yeti-web/.gem/bin/bundle exec rake pgq:worker processor=%i pidfile=/run/yeti-cdr-billing-%i/%i.pid config_file=/home/yeti-web/config/cdr_billing.yml
+PIDFile=/run/yeti-cdr-billing-%i/%i.pid
```
Comment (dmitry-sinina) [inline debian/yeti-web.yeti-cdr-billing@.service]: It won't work. May be it is systemd bug, but "PIDFile=/run/yeti-cdr-billing-%i/%i.pid" is not working.

**Comment by max-konin** [comment]
About system API. 

If this is used only for cron tasks, the best way is to using whenever gem (https://github.com/javan/whenever) and rake tasks...

**Comment by dmitry-sinina** [comment]
rake tasks will load application from scratch every time. Its is not so fast - we don't want spent 1 minute in application loading when we need run 2 second task.

**Comment by max-konin** [comment]
@dmitry-sinina To avoid loading the rails, you can write a custom runner like this https://github.com/mperham/sidekiq/blob/master/examples/scheduling.rb#L42....

But ok. We'll remove auth....

**Pull request merged**